packages/demo-block/package.json

```
@@ -12,13 +13,13 @@
     "clean": "nx exec -- rm -rf lib tsconfig.tsbuildinfo",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
-    "start": "npm run build && concurrently \"npm run build:watch\" \"web-dev-server --config ../../server.config.mjs --app-index __demo__\"",
-    "test": "npm run build && node ../../scripts/tests/run.mjs  --package=demo-block",
+    "start": "npm run build && concurrently \"npm run build:watch\" \"web-dev-server --config ../../server.config.js --app-index __demo__\"",
```

`server.config.js` doesn't exist. Could you look into this?

---

Nice one! I think we totally missed this one, we don't seem to use`web-dev-server` to serve anymore. I'm going to replace it with `vite`

---

Done kub.

---

ip address access is quite handy.
```suggestion
    "start": "npm run build && concurrently \"npm run build:watch\" \"vite serve __demo__ --open --force --host\"",
```

---

I'm thinking that we should do so across all packages in another PR. Might be more cleaner that way.